bob-apps/bin/bobc.rs

```diff
@@ -113,3 +117,11 @@ fn get_key_value(matches: &'_ ArgMatches<'_>) -> u64 {
         .parse()
         .expect("key must be u64")
 }
+
+fn get_key(k: u64) -> Vec<u8> {
+    // TODO move to static
```

Comment:
Probably we can reach BOB_KEY_SIZE const from bob-backend from here?

Reply:
I make it public in `bob-common/build.rs`